lib/app/features/core/providers/video_player_provider.c.dart

```diff
@@ -85,6 +82,12 @@ class VideoController extends _$VideoController {
         }
 
         _activeController = controller;
+
+        ref.listen(globalMuteProvider, (previous, next) {
```

Comment:
Hide the `previous` name because you don't use it.
```suggestion
        ref.listen(globalMuteProvider, (_, next) {
```